libs/api/core/data-access/src/lib/api-core-data-access.module.ts

```diff
@@ -1,9 +1,10 @@
 import { ApiConfigDataAccessModule } from '@mogami/api/config/data-access'
+import { PrometheusModule } from '@willsoto/nestjs-prometheus'
 import { Module } from '@nestjs/common'
 import { ApiCoreDataAccessService } from './api-core-data-access.service'
 
 @Module({
-  imports: [ApiConfigDataAccessModule],
+  imports: [ApiConfigDataAccessModule, PrometheusModule.register()],
```

Comment:
As mentioned in #217 we want to register the `PrometheusModule` in `ApiCoreFeatureModule`, not here.